protected/migrations/m170821_053800_cera_clean.php

```diff
@@ -49,7 +69,12 @@ public function safeUp()
 	    $address->country_id = 15;
 	    $address->contact_id = $contact->id;
 	    $address->date_end = null;
-	    $address->save();
+        if (!$address->save()) {
+            throw new CDbException("Unable to save address");
+        }
+        else {
```

Comment:
What is the purpose of these else blocks? If $address->save() is successful, why save again?